### PR TITLE
Storage release notes

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

File: releases/2019-11/2019-11-python.md [hunk 1]
```diff
@@ -61,11 +61,12 @@ and [secret](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/keyva
 
 ### Storage
 
-- Added batching support in blobs.
-- Added support to set rehydrate priority in blobs.
-- Breaking changes in `PermissionModels` for blobs, queues and files.
-- Modified several methods in blobs to use more keyworrd-only arguments.
-- Other breaking changes in blobs, queues and files.
+- azure.storage.file is now azure.storage.fileshare
+- Replaced iterable behavior with chunk based behavior in blobs.
```
Comment (xiafu-msft) [inline releases/2019-11/2019-11-python.md]: Do we want to indicate the rename? Maybe say now it has similar interface as stream? Support using readinto and readall to download data

File: releases/2019-11/2019-11-python.md [hunk 2]
```diff
@@ -95,9 +96,9 @@ on Windows
...
+| azure-storage-blob | [package](https://pypi.org/project/azure-storage-blob/12.0.0) | [Readme.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-blob) | [Preview Documentation](https://aka.ms/azsdk-python-storage-blob-ref) | [History.md](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/storage/azure-storage-blob/HISTORY.md) | [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-blob/samples) |
+| azure-storage-file | [package](https://pypi.org/project/azure-storage-file-share/12.0.0b5) | [Readme.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-file) | [Preview Documentation](https://aka.ms/azsdk-python-storage-fileshare-ref) | [History.md](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-file/HISTORY.md) | [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-file-share/samples) |
```
Comment (xiafu-msft) [inline releases/2019-11/2019-11-python.md]: Maybe change package name here?

File: releases/2019-11/2019-11-python.md [hunk 3]
```diff
@@ -61,11 +61,12 @@ and [secret](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/keyva
 
 ### Storage
 
-- Added batching support in blobs.
-- Added support to set rehydrate priority in blobs.
-- Breaking changes in `PermissionModels` for blobs, queues and files.
-- Modified several methods in blobs to use more keyworrd-only arguments.
-- Other breaking changes in blobs, queues and files.
+- azure.storage.file is now azure.storage.fileshare
```
[the remark this answers is not in the document]
Comment (annatisch) [reply releases/2019-11/2019-11-python.md]: We should definitely indicate the the changes go deeper than just the namespace, however we probably don't need to list every change here as these notes are more of a summary.

File: releases/2019-11/2019-11-python.md (hunk 2 above)
Comment (rakshith91) [reply releases/2019-11/2019-11-python.md]: good catch :)

File: releases/2019-11/2019-11-python.md (hunk 1 above)
Comment (annatisch) [reply releases/2019-11/2019-11-python.md]: This change in download behaviour is in file-share as well.
I would also probably reword this to something more generic like:
"New API for stream download responses for blobs and files. This object is no longer iterable."

File: releases/2019-11/2019-11-python.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (rakshith91) [reply releases/2019-11/2019-11-python.md]: The package is renamed from `azure-storage-file` to `azure-storage-file-share` along with renamed client modules.
